core/app/assets/js/utils/utils.js

```diff
@@ -6,4 +6,15 @@ export default class Utils {
 
 		return `${ pixels * 0.0625 }rem`;
 	}
+
+	static arrayToClassName = ( array ) => {
+		return array.filter( ( classItem ) => '' !== classItem ).join( ' ' );
+	}
+
+	static stringToRemValues = ( string ) => {
```

Comment:
Where this method is used?

Reply:
It's used by many components. PR will be sent soon.